src/strings/strings-en.ts

```diff
@@ -1027,6 +1025,7 @@ export const strings = {
   UPLOAD_PHOTO: 'Upload Photo...',
   UPLOAD_PHOTOS: 'Upload Photo(s)',
   USED_UP: 'Used Up',
+  USER_NOTIFICATION_ACTION: 'To change the preference, go to “Settings > Organization”',
```

Comment:
can we make the last part an href?

Reply:
Any part of the notification is clickable and takes to organization page, but I could make it and href for the styles